utf8.c

```diff
@@ -87,6 +87,34 @@ static Janet jutf8_to_codepoints(int argc, Janet *argv) {
   return janet_wrap_array(a);
 }
 
+static Janet jutf8_from_codepoints(int argc, Janet *argv) {
+  janet_fixarity(argc, 1);
+  JanetView jv = janet_getindexed(argv, 0);
+  const Janet *c = jv.items;
+  utf8proc_ssize_t l = jv.len;
+  utf8proc_ssize_t sl = l * 4;
```

Comment:
This appears unused, did you intend to do the alloca with sl?

Reply:
Oh, I think I did. I will check and fix that.